components/service/glean/build.gradle

```diff
@@ -14,8 +14,8 @@ apply plugin: 'kotlin-android'
  * created during unit testing.
  * This uses a specific version of the schema identified by a git commit hash.
  */
-String GLEAN_PING_SCHEMA_GIT_HASH = "04e203c"
-String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
+String GLEAN_PING_SCHEMA_GIT_HASH = "175920c"
+String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mdboom/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
```

Comment:
We will need to move this back to the base after merging https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/260